Mathlib/Algebra/Field/TransferInstance.lean

```diff
@@ -0,0 +1,60 @@
+/-
+Copyright (c) 2018 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl
+-/
+import Mathlib.Algebra.Field.Basic
+import Mathlib.Algebra.Ring.TransferInstance
+
+/-!
+# Transfer algebraic structures across `Equiv`s
+
+This continues the pattern set in `Mathlib/Algebra/Group/TransferInstance.lean`.
+
+### Implementation details
+
+When adding new definitions that transfer type-classes across an equivalence, please use
+`abbrev`. See note [reducible non-instances].
```

Comment:
Let's drop this (throughout), better to just refer to the implementation details in the base file.